src/tests/test_cases/test.py

```diff
@@ -337,6 +337,16 @@ def oneliner_d102(): return
 def oneliner_withdoc(): """One liner"""
 
 
+def ignored_decorator(func):   # noqa: D400,D401,D415
+    """Runs something"""
+    func()
+    pass
+
+
+@ignored_decorator
```

Comment:
Can you please add a test case next to this one with a single line definition similar to this one that doesn't have an ignored decorator and raises errors and another one that has a single line definition with a docstring that doesn't raise any errors to check that everything is working as expected in this context?

Reply:
Good idea :)
Are the tests I added meet the criteria?